src/gui/CemuApp.cpp

```diff
@@ -70,6 +73,38 @@ void unused_translation_dummy()
 
 bool CemuApp::OnInit()
 {
+	fs::path user_data_path, config_path, cache_path, data_path;
+	auto standardPaths = wxStandardPaths::Get();
+#ifdef PORTABLE
+	fs::path exePath(standardPaths.GetExecutablePath().ToStdString());
+	user_data_path = config_path = cache_path = data_path = exePath.parent_path();
+#else
+	SetAppName("cemu");
```

Comment:
```suggestion
	SetAppName("Cemu");
```
Should this be capital?

Reply:
I'm not sure if it really matters, but I've seen apps on Linux use either `App` or `app` as the config folder name.
I just put `cemu` as a placeholder since wxWidgets sets the app name as the executable name.